scss/_base_media.scss

```diff
@@ -5,6 +5,7 @@
 
   img {
     border: 0;
+    border-radius: 2px;
```

Comment:
@kwm14 I'm concerned we don't want border radius on every image on every site. I feel that is very opinionated and likely to lead to a lot of overrides. I think it's fine if to style image patterns with border radius.

Reply:
@anthonydillon originally me and Spencer wanted consistency as we use border-radius on buttons, cards etc so that was the reasoning behind it.

Speaking with Spencer this morning we're happy to have border-radius as a style to add onto a default un-styled image.

Reply:
Given it's such a minimal visual tweak - Is anyone really gonna add a class to add 2px of radius to an image? I really doubt it. Either we should do it site-wide or not at all?